src/api/posts/src/storage.js

```diff
@@ -22,6 +22,25 @@ const createInvalidFeedKey = (id) => createFeedKey(id).concat(invalidSuffix);
 // "NirlSYranl" to "t:feed:NirlSYranl:delayed"
 const createDelayedFeedKey = (id) => createFeedKey(id).concat(delayedSuffix);
 
+const getFeedKeysUsingScanStream = (matchPattern) => {
+  const keysArray = new Set();
```

Comment:
Because this isn't an `Array`, let's not name it that.  `keys` is fine.

Reply:
Hi prof, thank you, I have made the change. Please review :)